Pyrado/pyrado/logger/experiment.py

```diff
@@ -291,11 +340,15 @@ def _split_path_at(path, keyword):
     return custom, common
 
 
-def ask_for_experiment(latest_only: bool = False):
+def ask_for_experiment(
+    latest_only: bool = False, max_display: int = 10, show_hparams: Optional[List[str]] = None
```

Comment:
please reaname show_hparams since the name suggests that this is a boolian. how about hparam_list?